phalcon/Filter/FilterFactory.zep

```diff
@@ -11,15 +11,14 @@
 namespace Phalcon\Filter;
 
 use Phalcon\Filter;
-use Phalcon\Filter\FilterInterface;
 
 class FilterFactory
 {
     /**
      * Returns a Locator object with all the helpers defined in anonynous
      * functions
      */
-    public function newInstance() -> <FilterInterface>
+    public function newInstance() -> <Filter>
```

Comment:
This should return a filter interface because you can write filters yourself and register them in the factory